infrastructure/lib/constructs/lambda-fn.ts

```diff
@@ -153,8 +153,11 @@ export class ApiFlexFunction extends cdk.Construct {
       timeout: cdk.Duration.seconds(5),
       ...props.functionPropsOverride,
       bundling: {
-        // forceDockerBundling: true,
+        forceDockerBundling: true,
         externalModules: ["pg-native", "atat-web-api-orm"],
+        environment: {
+          NODE_ENV: "production",
+        },
```

Comment:
Just curious as to why we added `NODE_ENV` here?

Reply:
🤷 I was testing a few different things and this one happened to stick around. It honestly probably doesn't matter since we don't specify `nodeModules` to any of the `esbuild` configs for the `NodeJsFunction` props.